### PR TITLE
Include changelog in package descrption

### PR DESCRIPTION
**Pull request by jogo**
Include CHANGES.rst in the long description, so users can see
what changed when visiting https://pypi.python.org/pypi/git-stacktrace

CHANGES.rst is manually maintained.

### CODE REVIEW

**Comment by jogo** [comment]
Looks like this needs work

**Comment by jparise** [comment]
Include changes in long_description too show it shows up on the PyPI page?

File: setup.cfg [hunk 1]
```diff
@@ -13,6 +13,7 @@ home-page = https://github.com/pinterest/git-stacktrace
 
 description-file =
```
Comment (jogo) [inline setup.cfg]: This updates the long description

Comment (jparise) [inline setup.cfg]: Ah, I see. `pbr` handles this?

Comment (jogo) [inline setup.cfg]: Yup, https://github.com/openstack-dev/pbr/blob/64699d79bec5f0955566aa9129a6b9dbf686035f/pbr/util.py#L304

**Pull request merged**